libseccomp/src/filter_context.rs

```diff
@@ -432,6 +493,16 @@ impl ScmpFilterContext {
     ///
     /// If this function is called with an invalid filter or an issue is
     /// encountered getting the current state, an error will be returned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use libseccomp::*;
+    /// let mut ctx = ScmpFilterContext::new_filter(ScmpAction::Allow)?;
+    /// ctx.set_ctl_nnp(true)?;
+    /// assert!(ctx.get_ctl_nnp()?);
```

Comment:
Since `true` is the default I would flip the logic:
```suggestion
    /// ctx.set_ctl_nnp(false)?;
    /// assert!(!ctx.get_ctl_nnp()?);
```

Reply:
Added the fix in [commit](https://github.com/libseccomp-rs/libseccomp-rs/commit/2e07e17f07f92e2b6f7766c3c2d67495dc762efe)